src/everest/config/simulator_config.py

```diff
@@ -91,20 +108,28 @@ class SimulatorConfig(BaseModelWithContextSupport, extra="forbid"):
         | None
     ) = Field(
         default=None,
-        description="Defines which queue system the everest submits jobs to",
+        description=dedent(
+            """
+            Defines which queue system the everest submits jobs to.
+            """
+        ),
         discriminator="name",
         validate_default=True,
     )
     resubmit_limit: NonNegativeInt = Field(
         default=1,
-        description="""
-        Defines how many times should the queue system retry a forward model.
-
-    A forward model may fail for reasons that are not due to the forward model
-    itself, like a node in the cluster crashing, network issues, etc.  Therefore, it
-    might make sense to resubmit a forward model in case it fails.
-    resumbit_limit defines the number of times we will resubmit a failing forward model.
-    If not specified, a default value of 1 will be used.""",
+        description=dedent(
+            """
+        Defines how many times the queue system may retry a forward model.
+
+        A forward model may fail for reasons that are not due to the forward
+        model itself, like a node in the cluster crashing, network issues, etc.
+        Therefore, it might make sense to resubmit a forward model in case it
+        fails. `resumbit_limit` defines the number of times we will resubmit a
+        failing forward model. If not specified, a default value of 1 will be
+        used.
```

Comment:
A forward model may fail for external reasons, examples include node in the cluster crashing, network issues, etc.